src/pynwb/tests/test_optogenetics.py

```diff
@@ -103,8 +105,7 @@ def test_roundtrip(self):
             assert read_epochs[0, "theta_filter_reference_ntrode"] == 1
             assert read_epochs[0, "spatial_filter_on"]
             assert read_epochs[0, "spatial_filter_lockout_period_in_samples"] == 10
-            assert all(read_epochs[0, "spatial_filter_bottom_left_coord_in_pixels"] == (260, 920))
-            assert all(read_epochs[0, "spatial_filter_top_right_coord_in_pixels"] == (800, 1050))
+            assert all(read_epochs[0, "spatial_filter_region_node_coordinates_in_pixels"][0] == (260, 920))
```

Comment:
```suggestion
            assert all(read_epochs[0, "spatial_filter_region_node_coordinates_in_pixels"] == ((260, 920), (800, 1050)))
```

I think this should work